solution/src/components/card/index.js

```diff
@@ -25,7 +25,7 @@ function Card() {
     public_repos,
     repos_url,
     login,
-    url,
+    url
```

Comment:
I don't think this change was necessary (I assume your Prettier config has trailing commas turned off)

```suggestion
    url,
```

Reply:
oh yeah that was definitely prettier not me eeks

Reply:
if you commit the suggestion @oliverjam made I can merge this :)

Reply:
@sofiapoh I have fixed the trailing comma, sorry it took so long, this got lost in my notifications